config/config_test.go

```diff
@@ -2,156 +2,59 @@ package config
 
 import (
 	"testing"
+	"time"
 
-	"github.com/anz-bank/sysl-go/common"
-	"github.com/anz-bank/sysl-go/validator"
-
+	"github.com/alecthomas/assert"
```

Comment:
Please replace with `github.com/alecthomas/assert` with `github.com/stretchr/testify/require` to keep consistent with other tests.